frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/activity-card/data-resiliency-activity.tsx

```diff
@@ -5,11 +5,11 @@ import { getResiliencyProgress } from '../../../../utils';
 
 export const isDataResiliencyActivity = (results: PrometheusResponse): boolean => {
   const progress = getResiliencyProgress(results);
-  return progress && progress < 100;
+  return Number(progress) < 100;
```

Comment:
@cloudbehl here `null < 100` would fail and hence used 'undefined`.

Reply:
```suggestion
  return progress < 100;
```
if you return plain `number | undefined` here it will all work fine.

Reply:
Well, I see its working fine always.
Anyways pushed some new changes with returning `number` from general function.